pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -173,6 +173,24 @@ func (ic *ifConfigurator) configureContainerLink(
 	return nil
 }
 
+func (ic *ifConfigurator) configureContainerMTU(
+	podName string,
+	podNameSpace string,
+	containerID string,
+	containerNetNS string,
+	containerIFDev string,
+	hostInterfaceName string,
+	mtu int,
+) error {
+	infraContainerID := getInfraContainer(containerID, containerNetNS)
```

Comment:
Hi @wenyingd , could you help to take a look at the change for interface_configuration_*.go first? thanks.